providers/implementations/rands/seeding/rand_unix.c

```diff
@@ -641,7 +641,7 @@ size_t ossl_pool_acquire_entropy(RAND_POOL *pool)
         /* Maximum allowed number of consecutive unsuccessful attempts */
         int attempts = 3;
 
-        bytes_needed = ossl_rand_pool_bytes_needed(pool, 1 /*entropy_factor*/);
+        bytes_needed = ossl_rand_pool_bytes_needed(pool, 8 /*entropy_factor*/);
```

Comment:
I assume this isn't meant to be part of the PR...

Reply:
Yep. See commit message of e0542c2408e4ad8d70952b5e576cd73028646875

Reply:
Did the rand test succeed on VMS?

Reply:
> dropme! rand: simulate a weak entropy source
>
> TODO: this should be implemented as a test somehow

@paulidale @slontis I wasn't able to find an existing test which tests seeding using a provided seed source... how could such an important feature get accepted without a unit test?  🧐 

Having a seeding test for  a strong and a weak entropy source would be a good thing to have, don't you think so?

Reply:
I haven't tested this on VMS yet, but will do next

Reply:
I am going to test this on NonStop. Does this require egd for an effective test?

Reply:
All LGTM on multiple tests in multiple configs.

Reply:
The `dropme!` commit has been dropped now.